### ISSUE TITLE
AsuraScans Bookmarking Problem

### ISSUE THREAD

**Issue by CerealKiller7** [null2264/yokai#2]
### Steps to reproduce

1. Open AsuraScans
2. Open a title of your choice
3. Let it load all the way
4. Add it to your library
5. Exit out of AsuraScans
6. The title reloads again as if was never saved in the library or cached and acts as it is a new title

### Expected behavior

A title should be added only once to the library and have the "In the library" bubble whenever opening and browsing an extension.

### Actual behavior

With this bug you can add multiple entries to your library of the same title.

### TachiyomiJ2K version

1.7.6

### Android version

Android 13

### Device

Samsung Galaxy a51

### Other details

This problem does not occur on the Tachiyomi app or the fork TachiyomiSY. It has been for many months a problem on the J2K fork which has never been solved.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I am reporting an issue exclusive to this fork. I have also checked that is not an issue on the [main version of Tachiyomi](https://github.com/tachiyomiorg/tachiyomi)
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] I have updated the app to version **[1.7.3](https://github.com/Jays2Kings/tachiyomiJ2K/releases/latest)**.
- [X] I have updated all installed extensions.
- [ ] I have admitted that I am a clown by having checked this box, as I have not read these acknowledgements.
- [X] I have filled out all of the requested information in this form.

**Comment by null2264** [comment]
You may need to migrate Asura -> Asura to fix your current library. I'll release the new version soon.

**Comment by CerealKiller7** [comment]
Migrating from Asura to Asura doesn't fix it. The titles do update when they are in the library, is just annoying because you have to go see which one has ben updated recently on the extension then go refresh the entrie on thhe library. While still operating as an extension, a little bit annoying and time consuming.

**Comment by null2264** [comment]
Strange it should work as intended now, can you double check you already in v1.7.7?

**Comment by CerealKiller7** [comment]
Just updated, the bug seems doesn't occur anymore, thanks :)
